src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--31">
+      <div class="calendar__day calendar__day--1 start-day-mon"></div>
```

Comment:
Each day div is missing the content for the day number. Consider adding the day number as content in each day div.